gef.py

```diff
@@ -5015,18 +5016,18 @@ def show_info_proc(self) -> None:
         info("Process Information")
         pid = gef.session.pid
         cmdline = self.get_cmdline_of(pid)
-        gef_print(f"\tPID {RIGHT_ARROW} {pid}")
-        gef_print(f"\tExecutable {RIGHT_ARROW} {self.get_process_path_of(pid)}")
-        gef_print(f"\tCommand line {RIGHT_ARROW} '{cmdline}'")
+        gef_print(f"\tPID {RIGHT_ARROW} {pid}",
+                  f"\tExecutable {RIGHT_ARROW} {self.get_process_path_of(pid)}",
+                  f"\tCommand line {RIGHT_ARROW} '{cmdline}'")
         return
 
     def show_ancestor(self) -> None:
         info("Parent Process Information")
         ppid = int(self.get_state_of(gef.session.pid)["PPid"])
         state = self.get_state_of(ppid)
         cmdline = self.get_cmdline_of(ppid)
-        gef_print(f"\tParent PID {RIGHT_ARROW} {state['Pid']}")
-        gef_print(f"\tCommand line {RIGHT_ARROW} '{cmdline}'")
+        gef_print(f"\tParent PID {RIGHT_ARROW} {state['Pid']}",
+                  f"\tCommand line {RIGHT_ARROW} '{cmdline}'")
```

Comment:
we could remove the comma and add an explicit `\n` here if we wanted to instead.

Reply:
Upon further reflection I don't like this behaviour: Separate args should be separated by `sep`, which is " ". Otherwise we are straying farther from `print`, which is already variadic.